src/EnergyPlus/HVACSizingSimulationManager.cc

```diff
@@ -0,0 +1,406 @@
+								InitConvTemp, PlantLoop( i ).FluidIndex, 
+								"createNewCoincidentPlantAnalysisObject" );
+				cp = GetSpecificHeatGlycol( PlantLoop( i ).FluidName, 
+								InitConvTemp, PlantLoop( i ).FluidIndex, 
+								"createNewCoincidentPlantAnalysisObject" );
+
+				PlantCoinicidentAnalysis tmpAnalysisObj ( // call constructor
+					PlantLoopName,
+					i,
+					PlantLoop( i ).LoopSide( SupplySide ).NodeNumIn,
+					density,
+					cp,
+					PlantSizData( PlantSizingIndex ).NumTimeStepsInAvg,
+					PlantSizingIndex );
+
+				plantCoincAnalyObjs.push_back( tmpAnalysisObj ); //store new object in vector
+			}
+		}
+	}
+
+	void HVACSizingSimulationManager::SetupSizingAnalyses()
+	{
+		using DataLoopNode::Node;
+		using DataPlant::PlantReport;
+		using DataSizing::PlantSizData;
+		using DataSizing::HeatingLoop;
+		using DataSizing::CoolingLoop;
+		using DataSizing::CondenserLoop;
+		using DataSizing::SteamLoop;
+
+		for ( auto &P : plantCoincAnalyObjs ) {
+			//call setup log routine for each coincident plant analysis object 
+			P.supplyInletNodeFlow_LogIndex = sizingLogger.SetupVariableSizingLog( 
+				Node(P.supplySideInletNodeNum).MassFlowRate,
+				P.numTimeStepsInAvg );
+			P.supplyInletNodeTemp_LogIndex = sizingLogger.SetupVariableSizingLog( 
+				Node(P.supplySideInletNodeNum).Temp,
+				P.numTimeStepsInAvg );
+			if ( PlantSizData(P.plantSizingIndex).LoopType == HeatingLoop 
+					|| PlantSizData(P.plantSizingIndex).LoopType == SteamLoop) {
+				P.loopDemand_LogIndex = sizingLogger.SetupVariableSizingLog( 
+					PlantReport(P.plantLoopIndex ).HeatingDemand,
+					P.numTimeStepsInAvg ) ;
+			} else if ( PlantSizData(P.plantSizingIndex).LoopType == CoolingLoop 
+						|| PlantSizData(P.plantSizingIndex).LoopType == CondenserLoop ) {
+				P.loopDemand_LogIndex = sizingLogger.SetupVariableSizingLog( 
+					PlantReport(P.plantLoopIndex ).CoolingDemand,
+					P.numTimeStepsInAvg ) ;
+			}
+
+		}
+	}
+
+	void HVACSizingSimulationManager::PostProcessLogs() 
+	{
+		//this function calls methods on log objects to do general processing on all the logged data in the framework
+		for ( auto &L : sizingLogger.logObjs ) {
+			L.AverageSysTimeSteps(); // collapse subtimestep data into zone step data
+			L.ProcessRunningAverage(); // apply zone step moving average
+		}
+	}
+
+	void HVACSizingSimulationManager::ProcessCoincidentPlantSizeAdjustments(
+		int const HVACSizingIterCount )
+	{
+
+		using namespace DataPlant;
+		using namespace PlantManager;
+		using namespace DataSizing;
+		using DataGlobals::FinalSizingHVACSizingSimIteration;
+
+		//first pass through coincident plant objects to check new sizes and see if more iteration needed
+		plantCoinAnalyRequestsAnotherIteration = false;
+		for ( auto &P : plantCoincAnalyObjs ) {
+			//step 1 find maximum flow rate on concurrent return temp and load
+			P.newFoundMassFlowRateTimeStamp = sizingLogger.logObjs[ P.supplyInletNodeFlow_LogIndex ].GetLogVariableDataMax( );
```

Comment:
What do the CI machines get here? This must be failing somehow to get the behavior.